### ISSUE TITLE
inventory hud improvements

### ISSUE THREAD

**Issue by dawnwin** [THEREALWWEFAN231/Inertia#959]
I use inertia a lot. I really like inventory on hud. But I'd like it can scale-able and change colours such  red(inertia's colour),....if i make it transparent on hud. My inventory also do. I really hate it

**Comment by feliscatusmeows** [comment]
Yes make hud more scalable. Mabye rename the color manager to customization and move it there? idk
